pom.xml

```diff
@@ -85,9 +85,21 @@
     <dependencies>
         <dependency>
             <groupId>com.exasol</groupId>
-            <artifactId>virtual-schema-common-java</artifactId>
-            <version>${vscommon.version}</version>
+            <artifactId>virtual-schema-common-document</artifactId>
+            <version>${vs-common-document.version}</version>
         </dependency>
+        <dependency>
+            <groupId>com.exasol</groupId>
+            <artifactId>virtual-schema-common-document</artifactId>
+            <version>${vs-common-document.version}</version>
+            <type>test-jar</type>
+            <scope>test</scope>
```

Comment:
Could you please move this to below `<-- Test dependencies -->` or `<-- Integration test dependecies -->`?